pkgs/development/libraries/gpgme/default.nix

```diff
@@ -30,9 +32,6 @@ stdenv.mkDerivation rec {
   NIX_CFLAGS_COMPILE =
     with stdenv; lib.optional (system == "i686-linux") "-D_FILE_OFFSET_BITS=64";
 
-  AM_CXXFLAGS =
-    with stdenv; lib.optional (isDarwin) "-D_POSIX_C_SOURCE=200809L";
-
```

Comment:
BTW: Why did you drop this? Should I include that change for 1.9.0? I'm actually a bit confused because this is for macOS/Darwin only and according to you PR you didn't build this on macOS - are you sure that works?